### PR TITLE
Bower assets

### PR DESCRIPTION
**Pull request by tarleb**
Install leaflet files via composer's bower-asset plug-in.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,11 +12,12 @@
         }
     ],
     "require": {
-        "yiisoft/yii2": "*"
+        "yiisoft/yii2": "*",
```
Comment (tonydspaniard) [inline composer.json]: Should we add one of the stable versions now?

Comment (tarleb) [inline composer.json]: Probably wouldn't hurt.  But then, composer's [`minimum-stability`](https://getcomposer.org/doc/04-schema.md#minimum-stability) is `stable` by default, so my guess is that it wouldn't make a difference in practice.

Comment (tonydspaniard) [inline composer.json]: Ok, just a bit concerned on the API changes of Yii2 (sometimes they are unexpected)

**Comment by tonydspaniard** [comment]
We should tell users that bower-assets extension must be installed (even though yii2 does that) what do you think?

**Comment by tarleb** [comment]
Antonio Ramirez <notifications@github.com> writes:

> We should tell users that bower-assets extension must be installed (even
> though yii2 does that) what do you think?

True, I forgot about that.  I'll add a note to the readme tomorrow morning.

**Comment by tonydspaniard** [comment]
Thanks!

**Pull request merged**